e2e/tests/start_call.spec.ts

```diff
@@ -451,7 +451,7 @@ test.describe('call post', () => {
 
         await page.getByTestId('PostDotMenu-Button-' + postID).click();
 
-        await page.locator('#CENTER_dropdown_' + postID).locator('li', {hasText: 'Edit'}).click();
+        await page.keyboard.type('e');
```

Comment:
What is this change about?

Reply:
This was to fix the e2e tests failing -- the tests were getting hung up because it couldn't find the edit button, so I changed it to use the keyboard shortcut.
I'm not sure if they fixed it since then, but that's what I found in the playwright playback above the [67879fd](https://github.com/mattermost/mattermost-plugin-calls/pull/741/commits/67879fd4a0d2f54d507c96c4d209616301d56ab9) fix, and locally.

Reply:
Cool, could we add a comment? Because the fact that we are typing the message right after makes this extra confusing.